markdown/en/ff-asn.md

```diff
@@ -127,12 +145,18 @@ The position of the `ff-asn-group-element` inside the `ff-asn-group` does not ma
     </ff-asn-group-element>
 </ff-asn-group>
 ```
-The `{{data-binding}}` for the `ff-asn-group-element` allows accessing both scopes - the `{{element}}` scope and the `{{group}}` scope. This is especially important if you want to display the unit which is configured on the server side.
+The `{{data-binding}}` for the `ff-asn-group-element` allows accessing
+both scopes - the `{{element}}` scope and the `{{group}}` scope. This is
+ especially important if you want to display the unit which is configured
+  on the server side.
```

Comment:
Leading spaces after line breaks aren't necessary, are they? Or are they supposed to serve a different purpose here?

Reply:
No, just sneaked their way in there when breaking the lines